library/src/main/kotlin/com/chuckerteam/chucker/internal/data/room/HttpTransactionDao.kt

```diff
@@ -22,8 +22,8 @@ internal interface HttpTransactionDao {
     @Query(
         "SELECT id, requestDate, tookMs, protocol, method, host, path, scheme, responseCode, " +
         "requestPayloadSize, responsePayloadSize, error, graphQLDetected, graphQlOperationName FROM " +
-            "transactions WHERE responseCode LIKE :codeQuery AND path LIKE :pathQuery " +
-            "ORDER BY requestDate DESC"
+            "transactions WHERE responseCode LIKE :codeQuery AND (path LIKE :pathQuery OR " +
+            "graphQlOperationName LIKE :pathQuery) ORDER BY requestDate DESC"
     )
     fun getFilteredTuples(codeQuery: String, pathQuery: String): LiveData<List<HttpTransactionTuple>>
```

Comment:
```suggestion
    fun getFilteredTuples(codeQuery: String, pathQuery: String, graphQlQuery: String = ""): LiveData<List<HttpTransactionTuple>>
```

Reply:
And at the call site please pass the same parameter twice, mentioning why.

Please also consider the empty case ("")